src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -58,5 +59,129 @@ private static void OnElevationChanged(DependencyObject dependencyObject, Depend
 		}
 
 		#endregion
+
+#if NETFX_CORE
+		public static Thickness GetPadding(this UIElement uiElement)
+		{
+			if (uiElement is FrameworkElement fe && fe.TryGetPadding(out var padding))
+			{
+				return padding;
+			}
+
+			var property = uiElement.GetDependencyPropertyUsingReflection("PaddingProperty");
+			return property == null ? default : (Thickness)uiElement.GetValue(property);
```

Comment:
This must be a safe-cast
```suggestion
			return property != null && uiElement.GetValue(property) is Thickness t ? t : default;
```